### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 37

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#213] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255094-ea512bef-6ed0-49c1-b536-fa1022f23fc3.png)

**Comment by i0nut02** [comment]
1.  (200 scelgo 4)
2. (100 scelgo 2)* (100 scelgo 2)   ----> non so se posso fare questa cosa
3. (200 scelgo 4) - (100 scelgo 4)
